tests/test_keys.py

```diff
@@ -737,9 +745,11 @@ def test_is_pem_private(self):
     # Test for a valid PEM string.
     private_pem = self.rsakey_dict['keyval']['private']
     private_pem_ec = self.ecdsakey_dict['keyval']['private']
+    private_pem_encrypted = KEYS.create_rsa_encrypted_pem(private_pem, "pw")
```

Comment:
Maybe name it `private_pem_encrypted_rsa` to make it clear that it's an RSA PEM, similar to `private_pem_ec`?